src/Nuget.PackageIndex/Abstractions/FileSystem.cs

```diff
@@ -31,5 +31,20 @@ public DateTime FileGetLastWriteTime(string fullPath)
         {
             return File.GetLastWriteTime(fullPath);
         }
+
+        public string FileReadAllText(string fullPath)
+        {
+            return File.ReadAllText(fullPath);
+        }
+
+        public void FileWriteAllText(string fullPath, string content)
+        {
+            File.WriteAllText(fullPath, content);
+        }
+
+        public void FileDelete(string fullPath)
+        {
+            File.Delete(fullPath);
```

Comment:
That is true and is expected that abstractions don't do any error handling and do only operation that is expected